spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'faker'
+
+FactoryBot.define do
+  factory :user do
+    sequence(:email) { |n| Faker::Internet.email + n.to_s }
```

Comment:
We normally use string interpolation. I think it's easier to read and you use less resources. 
```suggestion
    sequence(:email) { |n| "#{Faker::Internet.email}#{n}" }
```